github/ci/prow-deploy/files/orgs.yaml

```diff
@@ -270,6 +270,9 @@ orgs:
         description: This is a demonstration of how KubeVirt could be displayed in Cockpit
         has_projects: false
         has_wiki: false
+      common-instancetypes:
```

Comment:
I am wondering if name common-instance-types would be better :)

Reply:
hmmm the term isn't hyphenated or even separated in our API or user docs, the latter might be a mistake on my part given AWS does refer to their version as `instance types`. I'm on the fence about the additional hyphen here tbh but I'll let others chime in.

Reply:
I vote also for keeping all realted naming consistent with the API, which means using `common-instancetypes` here.